lib/classes/PluginManager.js

```diff
@@ -429,6 +430,29 @@ class PluginManager {
     this.getCommand(commandsArray);
   }
 
+  /**
+   * If the command has no use when operated in a working directory with no serverless
+   * configuration file, throw an error
+   */
+  validateServerlessConfigDependency(command) {
+    if (_.has(command, 'configDependent') && command.configDependent === true) {
+      return getServerlessConfigFile(process.cwd()).then((result) => {
+        const isRunInService = !!result;
+
+        if (!isRunInService) {
```

Comment:
Here and in print.js, the similar code could be simply:
```
if (!service) {
```
That seems fairly clear, the triple `!` seems unnecessary.  Maybe there's something I'm not seeing that you're trying to guard against?
Note that I also suggested `service` which here means a rename of `result` but in print.js implies a new position for the conditional (assuming my comment there is misled).

Reply:
Yeah - The !! & ! was from the initial implementation from `plugins/lib/validate.js` and I suspect the reasoning was that `getServerlessConfigFile` will return an empty string when it fails to lookup, and they were trying to explicitly cast to Boolean. I agree it is unnecessary, and have updated accordingly 👍